db/migrate/20201022204510_add_extra_races_to_patient.rb

```diff
@@ -0,0 +1,9 @@
+class AddExtraRacesToPatient < ActiveRecord::Migration[6.0]
+  def change
+    add_column :patients, :unknown, :boolean
+    add_column :patients, :other, :boolean
+    add_column :patients, :refused_to_answer, :boolean
+
+    add_column :patients, :races, :string, array: true, default: [].to_yaml
```

Comment:
According to [the docs on serialize](https://apidock.com/rails/ActiveRecord/AttributeMethods/Serialization/ClassMethods/serialize), empty array will be converted to null anyway so this can just be `nil`.